src/sst.rs

```diff
@@ -486,6 +491,7 @@ impl<'a> EncodedSsTableBuilder<'a> {
             filter_offset: filter_offset as u64,
             filter_len: filter_len as u64,
             compression_codec: self.compression_codec,
+            row_attributes: vec![RowAttribute::Flags],
```

Comment:
Is this supposed to be hardcoded? It seems like it should use a provided value if it's for the builder...

Reply:
😅 would've caught that in the next PR when I was wondering why `ts` wasn't getting piped in when enabled! thanks for the catch